index.html

```diff
@@ -172,31 +172,45 @@ <h4>Before You Go...</h4>
     <div class="row">
       <div class="col-sm-12 m-4">
         <form>
+         <div class="form-group">
```

Comment:
I hope you were able to understand how form-group works and how useful it can be from BS4. Creating a form straight off from html and css can get extremely difficult but luckily BS has our back!